src/main/java/com/questhelper/helpers/quests/zogreflesheaters/ZogreFleshEaters.java

```diff
@@ -271,15 +271,17 @@ public void setupSteps()
 		talkToZavistic = new NpcStep(this, NpcID.ZAVISTIC_RARVE, new WorldPoint(2598, 3087, 0), "Talk to Zavistic Rarve at the Yanille Wizards' Guild. If you don't have 66 Magic, ring the bell outside the guild.");
 		talkToZavistic.addDialogStep("I'm here about the sicks...err Zogres");
 
-		goUpToSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0), "Go upstairs to Sithik upstairs in north Yanille.");
+		goUpToSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0),
+			"Go upstairs to Sithik in a house north or the Wizards' Guild.");
 		talkToSith = new ObjectStep(this, NullObjectID.NULL_6887, new WorldPoint(2591, 3104, 1), "Talk to Sithik Ints in the bed to the west.");
 		talkToSith.addDialogStep("Do you mind if I look around?");
 		goUpToSith.addSubSteps(talkToSith);
 
 		searchWardrobe = new ObjectStep(this, ObjectID.WARDROBE_6877, new WorldPoint(2590, 3103, 1), "Search Sithik's wardrobe.");
 		searchCupboard = new ObjectStep(this, ObjectID.CUPBOARD_6876, new WorldPoint(2593, 3105, 1), "Search Sithik's cupboard.");
 
-		goUpToOgreSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0), "Go talk to Sithik upstairs in north Yanille.");
+		goUpToOgreSith = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2597, 3107, 0),
+			"Go upstairs to Sithik in a house north or the Wizards' Guild.");
```

Comment:
Applying the same suggestion as above
```suggestion
			"Go upstairs and speak to Sithik in the house north of the Wizards' Guild.");
```